p2p/p2p.proto

```diff
@@ -8,13 +8,16 @@ option java_outer_classname = "P2pProto";
 option java_package = "com.p2p";
 
 import "coredocument/coredocument.proto";
+import "google/protobuf/timestamp.proto";
 
 message Header {
   uint32 network_identifier = 1;
   string node_version = 2;
   bytes sender_id = 3;
   // Body message type
   string type = 4;
+  // sent timestamp
+  google.protobuf.Timestamp timestamp = 5;
```

Comment:
https://github.com/centrifuge/go-centrifuge/pull/820/files#r264037334

Reply:
Is this really needed? I don't think it adds anything. In the linked comment we should compare with the current time, not what is specified in the P2P message (that could be faked by the sender)